File: aeronave/src/main/java/br/com/teste/sonda/aeronave/entity/Aeronave.java
package br.com.teste.sonda.aeronave.entity;

import java.time.LocalDateTime;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Data
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Aeronave {

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Long id;
	private String nome;
	private String marca;
	private Long ano;
	private String descricao;
	private Boolean vendido;
	private LocalDateTime created;
	private LocalDateTime updated;
}

File: front-end/src/app/aeronave/aeronave.ts
export class Aeronave{
   id: number;
   nome: string;
   marca: string;
   ano: number;
   descricao: string;
   vendido: boolean;
   created: any;
   updated: any;

   constructor(nome: string, marca: string, ano: number, descricao: string, vendido: boolean){
      this.nome = nome;
      this.marca = marca;
      this.ano = ano;
      this.descricao = descricao;
      this.vendido = vendido;
  }
}